models/servicenow.yml

```diff
@@ -379,6 +382,18 @@ models:
       - name: is_change_request_unauthorized
         description: '{{ doc("is_change_request_unauthorized") }}'
 
+      - name: dv_priority_label
+        description: '{{ doc("label") }}'
+
+      - name: dv_impact_label
+        description: '{{ doc("label") }}'
+
+      - name: dv_urgency_label
+        description: '{{ doc("label") }}'
+
+      - name: dv_task_state_label
+        description: '{{ doc("label") }}'
```

Comment:
It could be confusing that all of these will have the same definition. Is there a way we can work the unique name into each of these field descriptions?

Reply:
This comment goes for all the other label descriptions as well.

Reply:
For example, I don't necessarily know what `dv_` stands for here.

Reply:
I don't know about listing separate definitions for each field since it'll be mostly redundant. Instead, I updated the definition for label to be "The display value, or human-readable text, for a **titular** choice option." That way the reader knows it's the label for the field of topic.

Also regarding `dv_`, that prefix was added by direct mention in the customer's request. I went back and forth on it since it's not very intuitive what it stands for. I did update the field description to mention it so hopefully it's clearer. I'm not opposed to just removing it entirely though.